daemons/controld/pacemaker-controld.h

```diff
@@ -28,9 +28,14 @@ typedef struct {
 
     //! Group of \p controld_flags values
     uint32_t flags;
+
+
+    // Mainloop and triggers
```

Comment:
The original plan was to keep triggers global. This gets undone in the next commit but I'll fix it after review.